### PR TITLE
OCPBUGS-26147: configure Snyk scanner

### PR DESCRIPTION
**Pull request by simonpasquier**

### CODE REVIEW

File: .snyk [hunk 1]
```diff
@@ -0,0 +1,9 @@
+exclude:
+  global:
+    - vendor/**
+    - '**/*_test.go'
+    - test/**
+ignore:
+  7a0d7c2d-7a92-4ac7-bd36-e2a71a5b8754:
```
Comment (simonpasquier) [inline .snyk:7]: this doesn't work :(

**Comment by jan--f** [comment]
@simonpasquier I suppose we can close this for now?

**Comment by simonpasquier** [comment]
@jan--f I've added an exception for cmd/prometheus-config-reloader/main.go (line 233) via the Snyk UI until we figure out how to do it via the .snyk file.

**Comment by simonpasquier** [comment]
I've also asked on the aos-devel mailing list for advice.

**Comment by jan--f** [comment]
Ack, should we have a comment in the `.snyk` file to make thing less confusing when we look at this again?

**Comment by simonpasquier** [comment]
/hold

waiting for inputs from other teams if they managed to make it work with the .snyk file. I'll put a comment if the only solution is to define individual exclusions in the UI.

**Review by simonpasquier — COMMENTED** [review]
@jan--f from the answers received, it doesn't seem to possible to configure individual exceptions apart from the Snyk UI. I've added a comment to the .snyk file.
I can squash the commits if you're happy with it.

**Comment by jan--f** [comment]
Sounds good, feel free to squash and I'll tag.

**Comment by simonpasquier** [comment]
/hold cancel
commits squashed

**Pull request merged**